.gitignore

```diff
@@ -28,6 +28,7 @@ classes/
 # Gradle bits from: https://github.com/github/gitignore/blob/master/Gradle.gitignore
 .gradle
 build/
+out/
```

Comment:
Where did this directory come from?

Reply:
That is created by IntelliJ IDEA instead of it using the `build/` stated by the `build.gradle`. That is one of the issues I've been having too.

Reply:
THE WORST.  Especially when I committed it before and it added a billion files.

Reply:
Reasonable addition then